plugins/plugin-chart-echarts/src/Gauge/transformProps.ts

```diff
@@ -127,7 +127,15 @@ export default function transformProps(chartProps: ChartProps) {
       fontSize: FONT_SIZE_MULTIPLIERS.detailFontSize * fontSize,
     },
   }));
-
+  const min = minVal || calculateMin(transformedData);
```

Comment:
Should we use `??` to replace `||` ？I'm not sure if it needs to be recalculated only when `minVal` is `null` instead of `0`

Reply:
I was not aware of the `??` operator. Exactly what is needed here. Thanks a lot!

Reply:
It's ok :). `??` is called [Nullish Coalescing](https://github.com/tc39/proposal-nullish-coalescing) which just for the case of `null` and `undefined` values. Here, if we use the `||` operator, it will also be calculated when the value of minVal is `0`, which is actually unnecessary.So it may be better to use nullish coalescing.